src/Components/Notes/NotesControl.test.jsx

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable require-await */
+import React from 'react'
+import {act, render, renderHook} from '@testing-library/react'
+import ShareMock from '../../ShareMock'
+import useStore from '../../store/useStore'
+import NotesControl from './NotesControl'
+import model from '../../__mocks__/MockModel.js'
+
+
+describe('Notes Control', () => {
```

Comment:
nit: I'm doing the actual name of the component under test, so `NotesControl` no space

Reply:
addressed.

Reply:
ok